test/ODataEndToEndTests/Microsoft.Restier.WebApi.Test.Services.TrippinInMemory/Models/Person.cs

```diff
@@ -17,6 +17,8 @@ public class Person
         [AutoExpand]
         public virtual ICollection<Person> Friends { get; set; }
 
+        public virtual ICollection<Trip> Trips { get; set; }
```

Comment:
Sure, as not null case I have tested with Friends, so I did not add, I will some.